README.md

```diff
@@ -1,52 +1,149 @@
 # The Poki Networking Library &nbsp;&nbsp;[<img src="https://img.shields.io/npm/v/@poki/netlib?color=lightgray">](https://www.npmjs.com/package/@poki/netlib)
 
 <img align="right" src="https://raw.githubusercontent.com/poki/netlib/main/.github/logo.png" width=140>
-The Poki Networking Library is a peer-to-peer library for web games, using WebRTC datachannels to provide UDP connections between players directly. Like the Steam Networking Library, but for web.  
-Netlib tries to make WebRTC as simple to use as the WebSocket interface (for game development).
+The Poki Networking Library is a powerful peer-to-peer networking solution for web games, leveraging WebRTC datachannels to enable direct UDP connections between players. Think of it as the Steam Networking Library for the web, designed to make WebRTC as simple to use as WebSockets for game development.
+
+<p></p>
+
+> ⚠️ **Warning**: This library is still under development and considered a beta. The library API can change. Make sure to get in touch if you want to go live with this so we can keep you up-to-date about changes.
```

Comment:
You can use this:
```markdown
> [!WARNING]  
> Critical content demanding immediate user attention due to potential risks.
```
Makes it a nice warning block.
[source](https://github.com/orgs/community/discussions/16925)